search_indices/views/base.py

```diff
@@ -85,5 +85,8 @@ def initial(self, request, *args, **kwargs):
             # Restrict querying information system queries to authenticated users.
             # The information system field contents are not public.
             self._filter_search_fields_for_unauthenticated()
+            self.attributes = FacetedAttributeBackend.get_attributes(
+                authenticated=False
+            )
```

Comment:
If it doesn't cause any issues, would be nice to have if/else branch here for both authenticated and unauthenticated. I got a bit paranoid about this somehow ending up overwriting the class attribute `BaseSearchDocumentViewSet.attributes`, but yeah that shouldn't happen of course because the instance copies the class attributes as instance attributes.